Bottom: eb9a53c3632e380a0c544fae34cecf36cb4aa8eb
Top:    638cb1b3c2ab4fce7bc13732127b4b89658470d1
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-01-20 09:17:04 +0300

Refresh of wr-base.patch

---

diff --git a/Makefile b/Makefile
index 08d99d2..53c3603 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 ifneq ($(KERNELRELEASE),)
     obj-m	:= netsrv.o netcli.o
     obj_ib-y	:= ib-sock.o ib-sock-util.o
-    obj_ib-y	+= ib-sock-mem.o ib-sock-ctl.o
+    obj_ib-y	+= ib-sock-mem.o ib-sock-ctl.o ib-sock-wr.o
     netsrv-objs	:= srv.o $(obj_ib-y)
     netcli-objs	:= cli.o $(obj_ib-y)
 else
